### PR TITLE
roachtest: change schema change test timeout to 4 hrs

### PR DESCRIPTION
**Pull request by vivekmenezes**
Also rename schema change tests to always have a schemachange
prefix.

We ought to also rename the test files (for later)

related to #31526

Release note: None

### CODE REVIEW

File: pkg/cmd/roachtest/create_index.go [hunk 1]
```diff
@@ -23,13 +23,13 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
 )
 
-func registerCreateIndexTPCC(r *registry) {
+func registerSchemaChangeTPCC(r *registry) {
```
Comment (petermattis) [inline pkg/cmd/roachtest/create_index.go]: Feels like this test should be moved to `schemachange.go`.

**Review by petermattis — APPROVED** [review]

File: pkg/cmd/roachtest/create_index.go (hunk 1 above)
Comment (vivekmenezes) [reply pkg/cmd/roachtest/create_index.go]: Done. Added a commit to move the test.

**Pull request merged**